### PR TITLE
Fix: Fixed issue where placeholders used the thumbnail instead of the icon

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- This fixes an issue where placeholders for Office Documents and other file formats were created based on the thumbnail instead of the icon.
- This PR also adds placeholders for images files

### CODE REVIEW

File: src/Files.App/ViewModels/UserControls/InfoPaneViewModel.cs
Comment (yaira2) [inline src/Files.App/ViewModels/UserControls/InfoPaneViewModel.cs:1]: I can split this into a separate PR if requested

File: src/Files.App/Data/Models/ItemViewModel.cs [hunk 1]
```diff
@@ -970,18 +969,23 @@ private async Task LoadItemThumbnailAsync(ListedItem item, uint thumbnailSize =
 				{
 					await dispatcherQueue.EnqueueOrInvokeAsync(async () =>
 					{
+						// Assign the thumbnail/icon to the listed item
 						item.FileImage = await iconInfo.IconData.ToBitmapAsync();
-						if (!string.IsNullOrEmpty(item.FileExtension) &&
-							!item.IsShortcut && !item.IsExecutable &&
```
Comment (hishitetsu) [inline src/Files.App/Data/Models/ItemViewModel.cs:975]: Should `!item.IsShortcut && !item.IsExecutable` be deleted?

File: src/Files.App/Data/Models/ItemViewModel.cs [hunk 2]
```diff
@@ -970,18 +969,23 @@ private async Task LoadItemThumbnailAsync(ListedItem item, uint thumbnailSize =
...
-							!ImagePreviewViewModel.ContainsExtension(item.FileExtension.ToLowerInvariant()))
+
+						// Add the file icon to the DefaultIcons list
+						if (!DefaultIcons.ContainsKey(item.FileExtension.ToLowerInvariant()) && !string.IsNullOrEmpty(item.FileExtension))
 						{
-							DefaultIcons.AddIfNotPresent(item.FileExtension.ToLowerInvariant(), item.FileImage);
+							var fileIcon = await FileThumbnailHelper.LoadIconAndOverlayAsync(item.ItemPath, thumbnailSize, false, true);
+							var bitmapImage = await fileIcon.IconData.ToBitmapAsync();
+							DefaultIcons.Add(item.FileExtension.ToLowerInvariant(), bitmapImage);
```
Comment (hishitetsu) [inline src/Files.App/Data/Models/ItemViewModel.cs:980]:
```suggestion
							DefaultIcons.TryAdd(item.FileExtension.ToLowerInvariant(), bitmapImage);
```
It should be considered that other threads may add first between `ContainsKey` and `Add`.

File: src/Files.App/Data/Models/ItemViewModel.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Data/Models/ItemViewModel.cs:975]: I'm not sure why there were excluded in the first place.

Comment (hishitetsu) [reply src/Files.App/Data/Models/ItemViewModel.cs:975]: Shortcuts and executables have their own icon, so if we include them, the default icon will be an icon for a specific shortcut or executable.

Comment (yaira2) [reply src/Files.App/Data/Models/ItemViewModel.cs:975]: I'll add it back

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**